instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/__init__.py

```diff
@@ -262,9 +266,22 @@ def _instrument(self, **kwargs):
             self._wrapped_function_name,
         ) = lambda_handler.rsplit(".", 1)
 
+        flush_timeout_env = os.environ.get(
+            OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT, ""
+        )
+        flush_timeout = 30000
+        try:
+            flush_timeout = int(flush_timeout_env)
```

Comment:
Just pass-by thinking that if gave a default in the function interface as `flush_timeout=3000` then you could something like this:

```suggestion
            flush_timeout = int(
                os.environ.get(
                    OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT, flush_timeout
                )
            )
```

Not saying it is necessarily easier to read, but just another way to write it! The log statement would have to be updated to fetch from the env again.

Reply:
Thank, let's leave it as it is - IMO reading flow is good now and clearly separates types (string vs int).